components/donation/Donation.component.tsx

```diff
@@ -61,7 +61,15 @@ export default function DonationView({ nodeInfo, show }: ParentProps) {
 
   const resp = useSWR("/api/offer", fetcher);
 
-  let bolt12: string = resp.data != undefined ? resp.data.offer.bolt12 : null;
+  let bolt12: string =
+    resp.data != undefined ? resp.data.offer[0].bolt12 : null;
+
+  const commandFetchInvoice =
+    "fetchinvoice " +
+    bolt12 +
+    " [msatoshi] [quantity] [recurrence_counter] [recurrence_start] [recurrence_label] [timeout] [payer_note] \n";
+  const commandPay =
+    "pay bolt11 [msatoshi] [label] [riskfactor] [maxfeepercent] [retry_for] [maxdelay] [exemptfee] [localofferid] [exclude] [maxfee] [description]";
```

Comment:
Same here, for the moment we assume that we have the bolt11 but this is somethings that we need to look at

` const commandPay ="pay {invoice}"`